### ISSUE TITLE
How is the op/second calculated?

### ISSUE THREAD

**Issue by nebrelbug** [dondido/express-es6-template-engine#11]
Hi! This project is really cool. A quick question: how do you define and calculate op/second? Is it how many times that template could be compiled in a second...? Thanks!

**Comment by nebrelbug** [comment]
Also, how exactly does this module work? My understanding so far is that it creates a new file with the template string, but the code is kind of hard to read: is this correct?

**Comment by dondido** [comment]
@nebrelbug Yes this is exactly right, op/s is the number of compilation per seconds. It does not create a new file with the template string as my intention was to keep performance as high as possible and not to add any overhead.

**Comment by Subwater** [comment]
How does you code parse the `$`

**Comment by nebrelbug** [comment]
@dondido so it's just returning a new Function()?

**Comment by nebrelbug** [comment]
This just ruined my day... I've been working over the last while on a template engine called [Squirrelly](https://www.npmjs.com/package/squirrelly) but this is faster, can do more, and takes less code.

Now I'm having a package existential crisis... do you think there's even any place for packages like mine with this code around? (Don't answer truthfully)

**Comment by dondido** [comment]
My honest opinion on this is that it doesn't matter how powerful and performant your template engine is. You can see that other engines are far more popular than `express-es6-template-engine` is no matter how many the op/s are. One has to consider the target market if wants to maximise audience reach.  Most developers rely on a mix of social, owned or even paid platforms for distribution of the content. So no matter how good your code is, if you don't shout out loud it will go unnoticed.

**Comment by nebrelbug** [comment]
@dondido could you post a link to the benchmark you ran? Is it on JSPerf
